src/index.html

```diff
@@ -7,21 +7,396 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Catalog</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
```

Comment:
You should link a CSS file in your HTML, not SCSS. SCSS is a preprocessor and browsers can't interpret it directly. You should compile your SCSS files into CSS and then include the resulting CSS file in your HTML.